core/kazoo_number_manager/src/kazoo_number_manager_maintenance.erl

```diff
@@ -98,7 +98,7 @@ update_number_services_view(?MATCH_ACCOUNT_ENCODED(_)=AccountDb) ->
                                  ]
                                 ,View
                                 ),
-    case kz_json:are_identical(View, NewView) of
+    case kz_json:are_equal(View, NewView) of
```

Comment:
@fenollp If view is pulled from the DB then it will have _rev whereas NewView would not.  Please make sure this is ignored when checking if equal otherwise they never will be.